pyfr/plugins/base.py

```diff
@@ -178,6 +195,20 @@ def __init__(self, intg, *args, **kwargs):
 
 
 class SurfaceMixin:
+    def _surf_region(self, intg):
+        # Parse the region
+        sidxs = surface_data(self.cfg, self.cfgsect, intg.system.mesh,
+                             intg.rallocs)
+
+        # Generate the appropriate metadata arrays
+        self._ele_surface, self._ele_surface_data = [], {}
```

Comment:
Return these as a tuple?

Reply:
Return what as a tuple? This init is almost identical to the region mixini init

Reply:
Have surf region return rather than set members.  It is more flexible and intuitive (call a function, get stuff back, rather than it implicitly setting members you need to query).